.pre-commit-config.yaml

```diff
@@ -0,0 +1,22 @@
+# See https://pre-commit.com for more information
+repos:
+  - repo: https://github.com/PyCQA/flake8
+    rev: 6.0.0
+    hooks:
+      - id: flake8
+        # Required to make flake8 read from pyproject.toml for now :(
+        additional_dependencies: ["flake8-pyproject"]
+        language_version: "3.10"
+  - repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black
+        language_version: "3.10"
+  - repo: https://github.com/psf/black
+    rev: 23.3.0
+    hooks:
+      - id: black
+        alias: black-check
+        name: black-check
+        language_version: "3.10"
```

Comment:
Is `language_version` really required for all options? In my [example config](https://wazo-dev.atlassian.net/browse/WAZO-2982) I put it for mypy as it was the only one that really seemed to require it, but if you want to require it for consistency you can use the option at the root level instead.

Reply:
No I wasn't sure if it was required, but thought it better to ensure consistency. You're right, root level would be better.